CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 <a name="Pending Release"></a>
 ## [Pending Release](https://github.com/lightstep/otel-launcher-java/compare/1.5.0...master)
 
+<a name="1.6.0"></a>
```

Comment:
This is ok, although usually goes into the "Pending Release" section, in case we don't merge soon.

Reply:
So you mean this should change only after merge?